tests/test_decryption_mediator.py

```diff
@@ -454,6 +475,95 @@ def test_decrypt_selection_all_present(self):
             self.expected_plaintext_tally[first_selection.object_id], result.tally
         )
 
+    def test_decrypt_ballot_compensate_all_guardians_present(self):
```

Comment:
I felt a little weird putting these in the decryption mediator, but there was precedent for tests that directly exercise the decryption functions.

I think it would be ideal in the longer term to pull out some of the shared setup into shared test helpers, so we could move these out.

Reply:
have you met the [ElectionguardTest](https://github.com/microsoft/electionguard-python/tree/main/src/electionguardtest) module?